src/test/java/seedu/address/model/order/PriceTest.java

```diff
@@ -37,18 +37,18 @@ void constructor_invalidPriceZero_throwsIllegalArgumentException() {
     @Test
     void isValidPrice() {
 
-        assertThrows(NumberFormatException.class, () -> Price.isValidPrice(null));
+        assertThrows(NullPointerException.class, () -> Price.isValidPrice(null));
         assertThrows(NumberFormatException.class, () -> Price.isValidPrice(""));
 
         assertFalse(Price.isValidPrice("-1"));
 
         assertThrows(NumberFormatException.class, () -> Price.isValidPrice(" "));
         assertFalse(Price.isValidPrice("0"));
-        assertThrows(NumberFormatException.class, () -> Price.isValidPrice("1.5"));
-        assertThrows(NumberFormatException.class, () -> Price.isValidPrice("1.0"));
 
         assertTrue(Price.isValidPrice("1"));
         assertTrue(Price.isValidPrice("100"));
+        assertTrue(Price.isValidPrice("1.5"));
```

Comment:
Could you add some test cases for 3 decimal places?

Reply:
you are right, I will include that. Thank you for mentioning

Reply:
Thank you for pointing out. I will fix it and push again.